lib/game.rb

```diff
@@ -0,0 +1,162 @@
+#game class file
+require './lib/cell'
+require './lib/board'
+require './lib/ship'
+
+ class Game
+
+  attr_reader :player, :cruiser, :submarine
+  attr_accessor :computer_board
+
+  def initialize(player)
+    @player = player
+    @computer_board = Board.new
+    @player_board = Board.new
+    @cruiser = Ship.new("Cruiser", 3)
+    @submarine = Ship.new("Submarine", 2)
+
+  end
+
+  def user_places_ships(ship, coordinates)
+    loop do
+      p "which ship would you like to place? Choose either cruiser or submarine. q to quit"
+      ship = gets.chomp.downcase
+
```

Comment:
I created the if statement since the user input was being passed into the `valid_placement?` method as a string and would cause an error. The if statement just matches the name of the ship to ship object and then passes that into the `valid_placement?` method.